ux.symfony.com/assets/controllers/style-switch-controller.js

```diff
@@ -0,0 +1,26 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
```

Comment:
I can't see where this is used - am I missing it?

Reply:
Hmm weird i remember dropping that... did i forgot to push some commit AS USUAL ? :(